spot-ml/src/main/scala/org/apache/spot/netflow/FlowSuspiciousConnectsAnalysis.scala

```diff
@@ -108,8 +93,6 @@ object FlowSuspiciousConnectsAnalysis {
 
     inputFlowRecords
       .filter(cleanFlowRecordsFilter)
```

Comment:
as we discussed on the phone, as we move towards more of a library of routines that people can compose for their own experiments, it is natural that people will want to execute the suspicious connects analysis on a data frame in a such way that it acts like an "add columns" operation and does not drop a bunch of columns (which might contain useful side information, like class labels) just because they are not consumed by the suspicious connects model